.gitignore

```diff
@@ -30,10 +30,8 @@ hs_err_pid*
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 
 # User-specific stuff:
-.idea/workspace.xml
 .idea/tasks.xml
```

Comment:
All those files can be deleted. I see you added `.idea/` at the end of the file.